plugins/jobs/web_client/templates/jobs_checkBoxMenu.jade

```diff
@@ -0,0 +1,11 @@
+a.g-job-checkbox-menu-link(data-toggle="dropdown", data-target="#g-job-checkbox-menu-#{title}")
+      | #{title}
+      i.icon-down-dir
```

Comment:
I had a slight preference also--my real concern was that a filter could be on and the user forget about that, then not find something they were looking for.  I don't think changing the icon gets around that problem, but my hope was it helped a bit.